src/taipy/core/scenario/_scenario_manager.py

```diff
@@ -127,15 +124,19 @@ def _create(
             if config.additional_data_node_configs
             else {}
         )
+
         pipelines = {}
+        tasks_and_config_id_maps = {task.config_id: task for task in tasks}
         for sequence_name, sequence_task_configs in config.sequences.items():
```

Comment:
We've already used the term `sequence` in the Scenario config? I did not know that

Reply:
yep, but only in ScenarioConfig, so we will need to update it on other places as well after this ticket